File-Formats/PDF/Working-with-DigitalSignature.md

```diff
@@ -3982,4 +3982,67 @@ document.Close(True)
 
 {% endtabs %}
 
-You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/Digital%20Signature/Retrieve-revocation-certificate-information-from-digital-signature-embed-timestamp).
\ No newline at end of file
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/Digital%20Signature/Retrieve-revocation-certificate-information-from-digital-signature-embed-timestamp).
+
+## Get images from the existing signed signature field
+
+You can get the images from the existing signed signature field’s appearance using the **GetImages** method in [PdfLoadedSignatureField](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedSignatureField.html) class. The following code example explains this.
```

Comment:
To obtain images from the appearance of an existing signed signature field, you can utilize the GetImages method within the PdfLoadedSignatureField class. The code example below provides a clear demonstration of this functionality.

Reply:
changed